setup1-contributor/src/commands/contribute.rs

```diff
@@ -573,18 +578,33 @@ impl Contribute {
                         #[cfg(feature = "azure")]
                         upload_file_to_azure_async(&self.response_filename, &upload_url).await?;
                         #[cfg(not(feature = "azure"))]
-                        self.upload_response(&upload_url, signature_and_response_file_bytes, auth_rng)
-                            .await?;
+                        self.upload_response(
+                            lock_response.response_chunk_id,
+                            lock_response.response_contribution_id,
+                            signature_and_response_file_bytes,
+                            auth_rng,
+                        )
+                        .await?;
                     } else {
-                        self.upload_response(&upload_url, signature_and_response_file_bytes, auth_rng)
-                            .await?;
+                        self.upload_response(
+                            lock_response.response_chunk_id,
+                            lock_response.response_contribution_id,
+                            signature_and_response_file_bytes,
+                            auth_rng,
+                        )
+                        .await?;
                     }
                 }
                 #[cfg(feature = "azure")]
                 UploadMode::Azure => upload_file_to_azure_async(&self.response_filename, &upload_url).await?,
```

Comment:
we should probably go ahead and remove the Azure stuff at some point

Reply:
Should we put that in a separate issue?

Reply:
it's a good idea